android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboardJSHandler.java

```diff
@@ -62,6 +64,21 @@ public int getKeyboardWidth() {
     return kbWidth;
   }
 
+  @JavascriptInterface
+  public String initialKeyboard() {
```

Comment:
If we wanted to adopt something kinda similar for iOS:

https://developer.apple.com/documentation/webkit/wkscriptmessagehandlerwithreply

We could then `reply` with a stub string like the one this method builds, passing it through that Swift function's 'reply' callback for use and interpretation.  (See the other comment in this batch.)

But... this is only available with iOS 14.0+.